Signal/src/call/PeerConnectionClient.swift

```diff
@@ -121,13 +145,11 @@ class PeerConnectionClient: NSObject, RTCPeerConnectionDelegate, RTCDataChannelD
     // RTCVideoTrack is fragile and prone to throwing exceptions and/or
     // causing deadlock in its destructor.  Therefore we take great care
     // with this property.
-    //
-    // We synchronize access to this property and ensure that we never
-    // set or use a strong reference to the remote video track if 
-    // peerConnection is nil.
```

Comment:
We were previously (trying to) synchronize access to `remoteVideoTrack`.  This isn't necessary; we only access `remoteVideoTrack` on the main thread.